library/options.inc.php

```diff
@@ -373,7 +373,7 @@ function generate_form_field($frow, $currvalue) {
       " value='" . $formDate . "' $disabled />";
     if (!$disabled) {
        $selector = "#form_{$field_id}";
-       $date_init .= "$('" . $selector . "').datetimepicker({timepicker: false, format:'" . $DateFormat . "'});";
+       $date_init .= "$('" . $selector . "').datetimepicker({timepicker: false,  format:'" . $DateFormat . "'});";
```

Comment:
Yes, that is indeed the case.
I would like to get three other pull requests merged before this one just based on conflicts.  Last night a did a test merge of all the pull requests, and there is a specific series that can be merged without causing large issues.  We are developing a back-log at the moment, and we need to get things organized a bit.
Good call.